daemon/errors.go

```diff
@@ -342,6 +342,8 @@ func errToResponse(err error, snaps []string, fallback errorResponder, format st
 			snapName = err.Snap
 		case *snapstate.InsufficientSpaceError:
 			return InsufficientSpace(err)
+		case *snapstate.BusySnapError:
+			kind = client.ErrorKindBusySnap
```

Comment:
I think you can fill snapName, `snapName = err.SnapInfo.InstanceName()`.

Would it be useful for the desktop to know which of the applications carried by the snap are running and what are their PIDs?

Reply:
I agree that at least we should fill snapName, the other information might also be useful depending on the usecase.